source/codegen/metadata/nidigital/attributes.py

```diff
@@ -763,134 +364,55 @@
-        },
         'name': 'START_TRIGGER_TERMINAL_NAME',
         'resettable': False,
         'type': 'ViString'
     },
     1150040: {
         'access': 'read only',
         'channel_based': True,
-        'documentation': {
-            'description': 'Specifies the terminal name from which the exported conditional jump trigger signal may be routed to other instruments through the PXI trigger bus. You can use this signal to trigger other instruments when the conditional jump trigger instance asserts on the digital pattern instrument.\n'
-        },
         'name': 'CONDITIONAL_JUMP_TRIGGER_TERMINAL_NAME',
         'resettable': False,
         'type': 'ViString'
     },
     1150041: {
         'access': 'read-write',
         'channel_based': True,
-        'documentation': {
-            'description': 'Specifies the destination terminal for exporting the Pattern Opcode Event. Terminals can be specified in one of two ways. If the digital pattern instrument is named Dev1 and your terminal is PXI_Trig0, you can specify the terminal with the fully qualified terminal name, /Dev1/PXI_Trig0, or with the shortened terminal name, PXI_Trig0.\n',
-            'table_body': [
-                [
-                    'PXI_Trig0',
-                    'PXI trigger line 0'
-                ],
-                [
-                    'PXI_Trig1',
-                    'PXI trigger line 1'
-                ],
-                [
-                    'PXI_Trig2',
-                    'PXI trigger line 2'
-                ],
-                [
-                    'PXI_Trig3',
-                    'PXI trigger line 3'
-                ],
-                [
-                    'PXI_Trig4',
-                    'PXI trigger line 4'
-                ],
-                [
-                    'PXI_Trig5',
-                    'PXI trigger line 5'
-                ],
-                [
-                    'PXI_Trig6',
-                    'PXI trigger line 6'
-                ],
-                [
-                    'PXI_Trig7',
-                    'PXI trigger line 7'
-                ]
-            ],
-            'table_header': [
-                'Defined Values:'
-            ]
-        },
         'name': 'EXPORTED_PATTERN_OPCODE_EVENT_OUTPUT_TERMINAL',
         'resettable': True,
         'type': 'ViString'
     },
     1150042: {
         'access': 'read only',
         'channel_based': True,
-        'documentation': {
-            'description': 'Specifies the terminal name for the output trigger signal of the specified instance of a Pattern Opcode Event. You can use this terminal name as an input signal source for another trigger.\n'
-        },
         'name': 'PATTERN_OPCODE_EVENT_TERMINAL_NAME',
         'resettable': False,
         'type': 'ViString'
     },
     1150043: {
         'access': 'read-write',
         'channel_based': False,
-        'enum': 'HistoryRamTriggerType',
```

Comment:
Did you mean to remove this `'enum'` from this attribute? I think that's why it was removed in the generated proto file.

Reply:
Good catch!

Reply:
fixed this